### PR TITLE
[other] Check event button solved.

### LINKED ISSUE #187: [other] Check event button not centered below 1000px.
### Actual Output:-
Screenshot 2022-12-10 095938

### Expected Output:-
Screenshot 2022-12-10 100634

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Rishav1707**
## Fixes Issue

This PR fixes the following issues:

closes #187 

## Changes proposed

Here comes all the changes proposed through this PR

- index.module.css file has been modified.
- Adding a padding property in the screen below 1000px.

## Check List (Check all the boxes which are applicable) 

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Before
Screenshot 2022-12-10 095938

## After
Screenshot 2022-12-10 100634

### CODE REVIEW

**Comment by Rishav1707** [comment]
Why have all the checks failed whenever I committed something? 

@apoorv-on-git @ashwinexe @kaiwalyakoparkar @AnuragThePathak

**Comment by Rishav1707** [comment]
Anyone is there for review??

**Comment by Rishav1707** [comment]
> Anyone is there for review??

Is anyone there?? @apoorv-on-git @kunal-kushwaha @ashwinexe @kaiwalyakoparkar

**Comment by ashwinexe** [comment]
Hey there the team is busy with changes with website, someone will review once they are available. You don't need to tag them unnecessarily. 

Appreciate your contributions!

**Comment by Rishav1707** [comment]
Hey @ashwinexe sorry for the inconvenience caused by me.

**Comment by AnuragThePathak** [comment]
@chandraprakash-darji would you like to review it once?

File: src/v2/components/common/check/index.module.css [hunk 1]
```diff
@@ -31,7 +31,7 @@
    
 }
 .events-link{
-    padding: 5px 15px 30px 10px;
+    padding: 5px 15px 31px 20px;
```
Comment (AnuragThePathak) [inline src/v2/components/common/check/index.module.css:34]: Does it look bad if 32px is used instead of 31?

Comment (Rishav1707) [reply src/v2/components/common/check/index.module.css:34]: No, just minor change it have

**Comment by Chandraprakash-Darji** [comment]
@Rishav1707 decrease letter spacing, make font normal not capitalize,

**Comment by Rishav1707** [comment]
> @Rishav1707 decrease letter spacing, make font normal not capitalize,

Done @Chandraprakash-Darji 

Done

**Comment by SuperAayush** [comment]
Hey @Rishav1707!!

This has already been fixed
Thank You for the effort and contribution.

Closing the PR.

**Pull request closed without merging**